zerver/webhooks/insping/doc.md

```diff
@@ -0,0 +1,46 @@
+Get Insping notifications in Zulip! Insping (stylized as !nsping) is a
+simple uptime and performance monitoring tool, which notifies you when
+a website or service is up and running or down.
+
+{!create-stream.md!}
+
+{!create-bot-construct-url.md!}
+
+{!append-stream-name.md!}
+
+{!append-topic.md!}
+
+1. Go to [Insping's website](https://www.insping.com). If you haven't created an
+   account, sign up for a free account.
+
+2. Create an organization in the
+   [manage section](https://app.insping.com/#manage)
+   if there isn't one and set an URL that you want to monitor. After it's done,
+   it should look something like this:
+   ![](/static/images/integrations/insping/001.png)
+
+3. Navigate to [the integrations](https://app.insping.com/#integrations) section:
+   ![](/static/images/integrations/insping/002.png)
+
+4. In the **Webhooks** section, press **Create Webhook**. A box will
+   show up:
+   ![](/static/images/integrations/insping/003.png)
+
+5. Recall your incoming webhook URL that you created. Put it in the
+   **Webhook URL** section.
```

Comment:
I think `In the **Webhook URL** section, provide the webhook URL constructed above:` is better! :)

Reply:
Changed based on suggestion (there's no pic so `.` is okay)